### DISCUSSION TITLE
Support `v1alpha` gRPC reflection

### DISCUSSION THREAD

**Discussion by ttkzhed** [hyperium/tonic#1784] [Q&A] [answered]
Hi, we just tried out tonic 0.12.0 and we noticed it's using `v1` reflection instead of `v1alpha` (related tonic change: https://github.com/hyperium/tonic/pull/1701).

That breaks usage of common gRPC tools like *Postman* and *Kreya*.

There doesn't seem a way to support `v1alpha` reflection anymore, are we missing anything?
Otherwise, would it make sense to support this use case via opt-in?

E.g.
```rust
tonic_reflection::server::Builder::configure()
  .enable_v1alpha_reflection()
  ...
  .build()
```

**Comment by ttkzhed** [comment]
There's a Pull Request with a potential fix: https://github.com/hyperium/tonic/pull/1787

**Reply by ttkzhed** [reply, marked as answer]
Fixed in https://github.com/hyperium/tonic/pull/1888!

**Comment by 4F2E4A2E** [comment]
@v0.12.3 using `build_v1alpha()` did it for me
